src/posts/announce-2.3.0.Final.adoc

```diff
@@ -0,0 +1,199 @@
+* [https://issues.jboss.org/browse/THORN-2249[THORN-2249]] make sure `mvn clean install` passes and there are no unexpected errors and warnings (Done)
+* [https://issues.jboss.org/browse/THORN-2250[THORN-2250]] make sure the standalone test suite passes and there are no unexpected errors and warnings (Done)
+* [https://issues.jboss.org/browse/THORN-2265[THORN-2265]] remove all usages of deprecated APIs after migration to WildFly 14 (Done)
+* [https://issues.jboss.org/browse/THORN-2269[THORN-2269]] the fraction plugin should allow sharing one module-rewrite.conf among multiple fractions (Done)
+* [https://issues.jboss.org/browse/THORN-2270[THORN-2270]] the fraction plugin should be able to remove an <artifact> from module.xml (Done)
+* [https://issues.jboss.org/browse/THORN-2271[THORN-2271]] add workaround for WFLY-11373 to our integration of SmallRye Fault Tolerance (Done)
+* [https://issues.jboss.org/browse/THORN-2272[THORN-2272]] fix how Infinispan caches are forced to be active (Done)
+* [https://issues.jboss.org/browse/THORN-2273[THORN-2273]] don't let RESTEasy provide their own MicroProfile RestClient (Done)
+* [https://issues.jboss.org/browse/THORN-2274[THORN-2274]] upgrade Keycloak to 4.6.0.Final (Done)
+* [https://issues.jboss.org/browse/THORN-2275[THORN-2275]] make sure `mvn clean install` in the Examples passes and there are no unexpected errors and warnings (Done)
+* [https://issues.jboss.org/browse/THORN-2282[THORN-2282]] upgrade Keycloak to 4.7.0.Final (Done)
+* [https://issues.jboss.org/browse/THORN-2294[THORN-2294]] upgrade Keycloak to 4.8.1.Final (Done)
+
+=== Library Upgrade
+* [https://issues.jboss.org/browse/THORN-2014[THORN-2014]] Update Hibernate to 5.2.16+ (Done)
+
+=== Component Upgrade
+* [https://issues.jboss.org/browse/THORN-2127[THORN-2127]] Update to SmallRye REST Client 1.1 (Done)
+* [https://issues.jboss.org/browse/THORN-2170[THORN-2170]] MicroProfile 2.1 (Done)
+* [https://issues.jboss.org/browse/THORN-2286[THORN-2286]] Update to SmallRye OpenTracing 1.2 (Done)
+* [https://issues.jboss.org/browse/THORN-2289[THORN-2289]] Update to WildFly 15 (Done)
+* [https://issues.jboss.org/browse/THORN-2298[THORN-2298]] update to WildFly 15.0.1 (Done)
+
+=== Documentation
+* [https://issues.jboss.org/browse/THORN-2290[THORN-2290]] Clarify default datasource (Done)
+* [https://issues.jboss.org/browse/THORN-2292[THORN-2292]] Environment Variable usage doc missing (Done)
+
+== Breaking changes
+* [https://issues.jboss.org/browse/THORN-2232[THORN-2232]] Remove Teiid and its related Fractions from main code repo of Thorntail
+
+== Resources
+
+Per usual, we tend to hang out on `irc.freenode.net` in `#thorntail`.
+
+All bug and feature-tracking is kept in http://issues.jboss.org/browse/THORN[JIRA].
+
+Examples are available in https://github.com/thorntail/thorntail-examples/tree/2.3.0.Final
+
+Documentation for this release is available:
+
+* link:http://docs.wildfly-swarm.io/2.3.0.Final/[Documentation]
```

Comment:
Not sure why this links to `wildfly-swarm.io`, should be `thorntail.io`.

Reply:
Fixed